hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/config/EnvConfig.java

```diff
@@ -0,0 +1,79 @@
+package io.securecodebox.persistence.config;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * This class provides an API to access configuration values from environment variables
+ */
+@Slf4j
+public final class EnvConfig {
+
+  public boolean isDev() {
+    return Boolean.parseBoolean(retrieveEnvVar(EnvVarNames.IS_DEV));
+  }
+
+  public String scanName() {
+    return retrieveEnvVar(EnvVarNames.SCAN_NAME);
+  }
+
+  public String namespace() {
+    return retrieveEnvVar(EnvVarNames.NAMESPACE);
+  }
+
+  public boolean lowPrivilegedMode() {
```

Comment:
@J12934 What's the purpose of this config?

Reply:
https://www.securecodebox.io/docs/hooks/defectdojo#low-privileged-mode